packages/calcite-components/src/components/time-picker/time-picker.e2e.ts

```diff
@@ -1122,7 +1234,7 @@ describe("calcite-time-picker", () => {
       expect(fractionalSecondEl.innerHTML).toEqual("00");
     });
 
-    it("upward nudge of empty fractional second sets to 000 for step=0.001", async () => {
+    it("upward nudge of empty fractional second sets to 000 for step=0.001.", async () => {
```

Comment:
Is this trailing period intentional? We typically don't use periods in test names.

Reply:
woopsie, forgot to remove that.  I added that just to trigger a diff for the test runner, but didn't intend to commit it.